frontend/src/bundles/friends/friends-api.ts

```diff
@@ -0,0 +1,98 @@
+import { ApiPath, ContentType } from '~/bundles/common/enums/enums.js';
+import { FriendsApiPath } from '~/bundles/friends/enums/enums.js';
+import { type Http } from '~/framework/http/http.js';
+import { BaseHttpApi } from '~/framework/http-api/http-api.js';
+import { type Storage } from '~/framework/storage/storage.js';
+
+import {
+    type FriendRequestDto,
+    type FriendResponseDto,
+    type PaginationParameters,
+} from './types/types.js';
+
+type Constructor = {
+    baseUrl: string;
+    http: Http;
+    storage: Storage;
+};
+
+class FriendsApi extends BaseHttpApi {
+    public constructor({ baseUrl, http, storage }: Constructor) {
+        super({ path: ApiPath.FRIENDS, baseUrl, http, storage });
+    }
+
+    public async getNotFollowed(payload: PaginationParameters): Promise<{
+        users: FriendResponseDto[];
+        query: PaginationParameters;
+    }> {
+        const { page, limit } = payload;
+        const response = await this.load(
+            this.getFullEndpoint(FriendsApiPath.ROOT, {}),
+            {
+                method: 'GET',
+                contentType: ContentType.JSON,
+                hasAuth: true,
+                query: { page, limit },
+            },
+        );
+
+        return await response.json<{
+            users: FriendResponseDto[];
+            query: PaginationParameters;
+        }>();
+    }
+
+    public async getFollowings(payload: PaginationParameters): Promise<{
+        users: FriendResponseDto[];
+        query: PaginationParameters;
+    }> {
```

Comment:
Can we change these return types to Paged<FriendResponseDto>?

Reply:
Yes, I fixed it.